test/TestUtils.py

```diff
@@ -160,23 +161,34 @@ def test_expand_paths_vars(monkeypatch):
 @pytest.mark.parametrize(
     ('reset_env_var', 'message_prefix'),
     (
-        ('PATH', 'Warning: Failed to locate command:'),
-        ('GIT_DIR', 'Warning: Failed to discover yaml files to lint using git:')
+        ('PATH',
+            "Failed to locate command: "),
+        ('GIT_DIR',
+            "Failed to discover yaml files to lint using git: fatal: not a git repository: ''")
     ),
     ids=('no Git installed', 'outside Git repository'),
 )
 def test_get_yaml_files_git_verbose(
     reset_env_var,
     message_prefix,
     monkeypatch,
-    capsys
+    caplog
 ):
     options = cli.get_config(['-v'])
+    utils.initialize_logger(options.verbosity)
```

Comment:
I think this should be put into some sort of a generator-based fixture that would reset the logging level on teardown.

Reply:
Read https://docs.pytest.org/en/latest/logging.html#caplog-fixture  which states that caplog fixture takes care of restoring logging setup after each test run.